CMakeLists.txt

```diff
@@ -7,7 +7,7 @@
 
 # CMAKE File for Albany building against an installed Trilinos
 
-cmake_minimum_required(VERSION 2.8.12)
```

Comment:
I can't believe we were still accepting CMake 2.X...

Reply:
Me neither :)